tests/js/results-tests.js

```diff
@@ -263,6 +263,10 @@ module.exports = {
 
     testResultsSortedAllTypes: function() {
         var realm = new Realm({schema: [schemas.BasicTypes]});
+        realm.write(() => { // FIXME: This should not be required!
+            realm.deleteAll();
```

Comment:
why we are deletingAll in a test which did not require this in the past. Seems strange. Can we investigate this.

Reply:
I got `clearTestState()` to work again.